### PR TITLE
Map Resize and Legend

### PR DESCRIPTION
**Pull request by ifarzana**

### CODE REVIEW

File: ui/src/core/xibo-cms.js [hunk 1]
```diff
@@ -3745,10 +3745,19 @@ function initJsTreeAjax(container, id, isForm, ttl)
...
+        if (typeof refreshDisplayMap === "function") {
+            refreshDisplayMap();
+        }
+    }
+
     reload = (typeof reload == 'undefined') ? true : reload;
     // Shrink table to ease animation
     if($('#grid-folder-filter').is(":hidden")) {
         $('#datatable-container').addClass('col-sm-10').removeClass('col-sm-12');
+        refreshDisplayMap();
```
Comment (maurofmferrao) [inline ui/src/core/xibo-cms.js:3760]: Shouldn't this one be `resizeDisplayMap()` instead?

File: ui/src/core/xibo-cms.js [hunk 2]
```diff
@@ -3760,6 +3769,7 @@ function adjustDatatableSize (reload) {
 
             // if the folder tree is hidden, then make it so datatable can take whole available width
             $('#datatable-container').addClass('col-sm-12').removeClass('col-sm-10');
+            refreshDisplayMap();
```
Comment (maurofmferrao) [inline ui/src/core/xibo-cms.js:3772]: Same as above comment

File: ui/src/core/xibo-cms.js (hunk 1 above)
Comment (ifarzana) [reply ui/src/core/xibo-cms.js:3760]: yeah, you are right, my mistake, thanks

**Review by maurofmferrao — DISMISSED** [review]
Looks good to me 👍🏻

**Review by maurofmferrao — COMMENTED** [review]
Looks good to me!

**Review by maurofmferrao — APPROVED** [review]

**Pull request merged**